rfcs/models/identity/identity-update-policy.md

```diff
@@ -0,0 +1,69 @@
+  "controller": "did:<iin_name>:<network_name>",
+  "type": "VerifiableCondition2021",
+  "conditionAnd": [{
+      "id": "did:<iin_name>:<network_name>#updatepolicy-1",
+      "controller": "did:<iin_name>:<network_name>",
+      "type": "VerifiableCondition2021",
+      "conditionOr": ["did:<iin_name>:<network_participant_3>#key1",
+        "did:<iin_name>:<network_participant_2>#key3"
+      ]
+    },
+    "did:<iin_name>:<network_participant_1>#key1"
+  ]
+}
+```
+
+## Network DID updates
+
+Any request to update the *Network DID* must be authenticated based on different policies for the different scenarios of (i) participants leaving the network, (ii) new participants joining the network, and (iii) other updates to the Network DID.
+
+### Participants leaving the network
+
+There can be two scenarios when an existing participant unit of a network leaves: (a) The participant itself willingly leaves the network. (b) Other participants of the network decide to expel that participant. Based on these two scenarios, any *Network DID* update request removing one participant from the `networkParticipants` property is authenticated as per the following conditions:
+
+A. The *Network DID* update request is attested by the participant who is leaving the network, that is the participant whose DID is being excluded from the  `networkParticipants` property. This indicates voluntary exit from the network.
+
+B. If condition A is not satisfied, and there exists an `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested satisfying the `updatePolicy`.
+
+C. If condition A is not satisfied and there is no `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested by strictly greater than half (`> N/3`) of the existing participants in the network indicated by `networkParticipants` property of the existing *Network DID* document.
+
+### Participants joining the network
+
+For new network participants joining the network, the DID update request must be authenticated based on the following conditions:
+
+A. If there is an `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested satisfying the `updatePolicy`, and it must also be attested by the participant which is being added to the `networkParticipants`.
+
+B. If there is no `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested by strictly greater than half (`> N/3`) of the existing participants in the network indicated by `networkParticipants` property of the existing *Network DID* document. The request must also be attested by the participant which is being added to the `networkParticipants` property.
+
+### Other updates to the Network DID
```

Comment:
would this include the policy for updating the update policies themselves? is such an update governed by the updated policy itself? are there potential edge cases here to consider?

Reply:
Satisfying the `updatePolicy` should indicate that the members who are governing the network have attested a request. So those requests should be allowed to update the `updatePolicy` itself as a network governance process. I gave it some thought but cannot think about any edge case at this moment. Let me know if you have found something.

Reply:
ok got it